ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_tensor.py

```diff
@@ -1487,6 +1487,41 @@ def test_paddle_cumprod(
     )
 
 
+# is_complex
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="paddle.to_tensor",
+    method_name="is_complex",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
```

Comment:
Arent you only generating real value inputs here? How are you going to test for complex values then 🤔

Reply:
Hello @YushaArif99 ,

Thanks for pointing this out, I have updated it.
Please could you take a look at it.